fjord/suggest/tests/test_sumosuggest.py

```diff
@@ -0,0 +1,189 @@
+import os
+
+from mock import MagicMock, patch
+from nose.tools import eq_, ok_
+import requests_mock
+
+from fjord.base.tests import TestCase, skip_if
+from fjord.feedback.tests import ResponseFactory
+from fjord.suggest.provider_sumosuggest import (
+    SUMO_SUGGEST_API_URL,
+    SUMOSuggestProvider
+)
+
+
+class SUMOSuggestProviderTestCase(TestCase):
+    provider = SUMOSuggestProvider()
+
+    def test_not_happy(self):
+        resp = ResponseFactory(
+            happy=True,  # Happy
```

Comment:
The naming of this test is inconsistent with the rest. Should it be `test_not_sad`?